src/lib.rs

```diff
@@ -16,12 +16,45 @@ use tracing_subscriber::{
     registry::LookupSpan,
 };
 
+const LINE_VERT: &str = "│";
+const LINE_HORIZ: &str = "─";
+const LINE_HORIZ_THICK: &str = "━";
+const LINE_BRANCH: &str = "┝";
```

Comment:
Slight nit: I think I'd prefer if the narrower variants were used everywhere if at all possible (e.g., https://graphemica.com/%E2%94%9C). Not sure how you feel about that!

Reply:
Oh, I see the conversation happening in #4. This might be the least non-terrible option, huh

Reply:
It looks like we should still use the two different versions of that character based on whether the branch is a span or an event, right? Looking at the screenshot, it looks like the thick branch character is used for both spans and events.

Reply:
Hm, after trying a bunch of variations I *think* I prefer the using the narrower variants slightly? It looks a bit cleaner and cleans up the code a bit as well. It's also what the `tree` and `cargo tree` commands do, so it seems pretty reasonable.